### PR TITLE
openscenegraph:Fix finding of SDL2 

### PR DESCRIPTION
**Pull request by BeagleJoe**
WIP
Fixed patchset to find sdl2
Disabled some dependencies -- builds now
Added CMAKE_BUILD_TYPE

### CODE REVIEW

**Comment by Begasus** [comment]
@BeagleJoe any progress on this? (I also see you have 2 separate recipes in here)

**Comment by BeagleJoe** [comment]
@Begasus I accidentally pushed a test speed_dreams recipe. I haven't figured out how to fix that (I'm still pretty new to git). 
I am inclined to just Close this PR and create a new one later.

File: games-engines/openscenegraph/openscenegraph-3.6.5.recipe [hunk 1]
```diff
@@ -111,14 +112,16 @@ BUILD_REQUIRES="
...
 	devel:libilmimf_2_4$secondaryArchSuffix
 	devel:libintl$secondaryArchSuffix
 	devel:libjasper$secondaryArchSuffix
 	devel:libjpeg$secondaryArchSuffix
 #	devel:libOpenCOLLADABaseUtils$secondaryArchSuffix
 	devel:libpng16$secondaryArchSuffix
 	devel:libpoppler${secondaryArchSuffix}_glib >= 8.12
-	devel:libsdl$secondaryArchSuffix
+	devel:libsdl_1.2$secondaryArchSuffix
+	devel:libSDL2_2.0$secondaryArchSuffix
```
[the remark this answers is not in the document]
Comment (BeagleJoe) [reply games-engines/openscenegraph/openscenegraph-3.6.5.recipe:124]: After further study, I'm not sure if either is needed. The only use of these that I can find is: One of the examples uses sdl1, but the examples are not built by the current recipe. On of the applications uses SDL2, but again the current recipe has BUILD_OSG_APPLICATIONS:BOOL=OFF

**Comment by mywave82** [comment]
@BeagleJoe you can edit commits that already have been pushed (for PR branches, this is normal)

`git rebase --interactive HEAD~2`

makes it possible to mark a commit with edit. You can go back in time and amend any commits (`git status` while doing this handy and provides the information needed to proceed at all time), and you can force push the branch back into github when happy.

**Pull request closed without merging**